### PR TITLE
add support for rust-1.51

### PR DESCRIPTION
**Pull request by michaelkirk**
See it in action: https://github.com/georust/geo/pull/646

### CODE REVIEW

File: rust-1.51/Makefile [hunk 1]
```diff
@@ -0,0 +1,74 @@
+GEORUST_ROOT=$(CURDIR)/../..
```
Comment (michaelkirk) [inline rust-1.51/Makefile:1]: I've had some flavor of this makefile floating around uncommitted for a while.

I'm not super proud of it, and if it's controversial, I can leave it out for now.

File: rust-1.51/proj-ci.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# https://hub.docker.com/orgs/georust/proj-ci
+
+FROM georust/libproj-builder:rust-1.50
```
Comment (michaelkirk) [inline rust-1.51/proj-ci.Dockerfile:3]:
```suggestion
FROM georust/libproj-builder:rust-1.51
```

Comment (michaelkirk) [reply rust-1.51/proj-ci.Dockerfile]: lol new levels of laziness allowed via GH suggestion self-comments.

File: rust-1.51/proj-ci.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# https://hub.docker.com/orgs/georust/proj-ci
+
+FROM georust/libproj-builder:rust-1.51
+
+RUN apt-get update \
+  && DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends \
+    clang \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY --from=georust/libproj-builder:rust-1.50 /build/usr /usr
```
Comment (michaelkirk) [inline rust-1.51/proj-ci.Dockerfile:10]:
```suggestion
COPY --from=georust/libproj-builder:rust-1.51 /build/usr /usr
```

File: rust-1.51/Makefile (hunk 1 above)
Comment (urschrei) [reply rust-1.51/Makefile:1]: Seems pretty sensible to me.

**Comment by urschrei** [comment]
LGTM? Let's merge if you're happy with it, so that the CI update cascade can begin…

**Pull request merged**